.github/workflows/ci.yml

```diff
@@ -39,3 +39,7 @@ jobs:
 
     - name: Firefox Test
       run: yarn test:browser --project=firefox
+
+    - name: Publish dev build
+      run: .github/scripts/publish-dev-build '${{ secrets.DEV_BUILD_GITHUB_TOKEN }}'
+      if: ${{ github.event_name == 'push' }}
```

Comment:
Is there a way to make this conditional on the pull request author's merge permissions? 

When I open a pull request that passes the rest of CI, I'll receive an email about this portion of the build failing due to lack of permissions.

Reply:
I guess we could make an own workflow for it which just listens for push events on `main`

Reply:
I opened https://github.com/hotwired/turbo/pull/851 for this